src/neo/Wallets/NEP6/NEP6Wallet.cs

```diff
@@ -300,5 +301,34 @@ public override bool VerifyPassword(string password)
                 }
             }
         }
+
+        public bool ChangePassword(string password_old, string password_new)
+        {
+            bool succeed = true;
+            lock (accounts)
+            {
+                Parallel.ForEach(accounts.Values, (account, state) =>
+                {
+                    if (!account.ChangePasswordPrepare(password_old, password_new))
+                    {
+                        state.Stop();
+                        succeed = false;
+                    }
+                });
+            }
+            if (succeed)
+            {
+                foreach (NEP6Account account in accounts.Values)
+                    account.ChangePasswordCommit();
```

Comment:
We don't check the result of this method, But I think that it's not needed so it's better to remove the return

Reply:
I suppose maybe we need the return value to tell whether the password is changed successfully?

Reply:
Have removed the return:)